app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -45,10 +40,15 @@ class MainActivity : AppCompatActivity() {
             val city = response.name
             val humidity = response.main.humidity.toString() + "%"
             val pressure = response.main.pressure
-            val tempRange = round((response.main.temp_min -273)*9/5 + 32).toString() + " to " + round((response.main.temp_max -273)*9/5 + 32).toString() + " Degrees"
-            Toast.makeText(this, tempRange, Toast.LENGTH_LONG).show()
+
+            //Formats temperature
+            fun formatText(temp:Float):String{
+                return round((temp -273)*9/5 + 32).toInt().toString()
+            }
+            
```

Comment:
I changed this to a method because it provides other developers easier readability and eliminates some redundancy in when called. Duplication is one of sonarQube's 7 deadly sins.

Reply:
Is it used in multiple places?  Or will it be used in multiple places?  If only used in one place, then moving one line of logic from an existing method to a new method does not reduce any duplication.